### PR TITLE
Enable cross-chain Coretime region transfers

### PR DESCRIPTION
**Pull request by Szegoo**
This PR allows Coretime regions to be transferable via XCM.

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5039054

**Review by joepetrowski — APPROVED** [review]

File: substrate/frame/broker/src/nonfungible_impl.rs [hunk 1]
```diff
@@ -50,3 +50,7 @@ impl<T: Config> Transfer<T::AccountId> for Pallet<T> {
 		Self::do_transfer((*index).into(), None, dest.clone()).map_err(Into::into)
 	}
 }
+
+// We don't allow any of the mutate operations, so the default implementation is used, which will
+// return `TokenError::Unsupported` in case any of the operations is called.
+impl<T: Config> Mutate<T::AccountId> for Pallet<T> {}
```
Comment (seadanda) [inline substrate/frame/broker/src/nonfungible_impl.rs:56]: Ideally there would be a simple test for this in case the defaults are changed

**Review by seadanda — COMMENTED** [review]
Looks good, just some housekeeping

File: substrate/frame/broker/src/nonfungible_impl.rs (hunk 1 above)
Comment (Szegoo) [reply substrate/frame/broker/src/nonfungible_impl.rs:56]: Added a test to cover this.

**Review by seadanda — APPROVED** [review]
Nice addition

**Pull request merged**